src/dispatcher.rs

```diff
@@ -571,11 +657,29 @@ pub extern "C" fn proxy_on_http_call_response(
     })
 }
 
+#[no_mangle]
+pub extern "C" fn proxy_on_grpc_receive_initial_metadata(
+    _context_id: u32,
+    token_id: u32,
+    headers: u32,
+) {
+    DISPATCHER.with(|dispatcher| dispatcher.on_grpc_receive_initial_metadata(token_id, headers))
+}
+
 #[no_mangle]
 pub extern "C" fn proxy_on_grpc_receive(_context_id: u32, token_id: u32, response_size: usize) {
     DISPATCHER.with(|dispatcher| dispatcher.on_grpc_receive(token_id, response_size))
 }
 
+#[no_mangle]
+pub extern "C" fn proxy_on_receive_grpc_trailing_metadata(
```

Comment:
This is still worng... It _has_ to be `proxy_on_grpc_receive_trailing_metadata` to match the signature from the host.

Please copy & paste this function name.

Reply:
oh. sorry...